tests/integration/incremental_schema_tests/test_incremental_schema.py

```diff
@@ -140,3 +140,4 @@ def test__bigquery__run_incremental_sync_all_columns(self):
     @use_profile('bigquery')
     def test__bigquery__run_incremental_fail_on_schema_change(self):
         self.run_incremental_fail_on_schema_change()
+     
```

Comment:
Nit: If there is no real change to this file and just the newline, can we get rid of this change so it doesn't show an edit happened in this file?

Reply:
I believe this is a pep8 thing, was also showing up in a lot of the pre-commit stuff i ran earlier as well. I can remove the line then possibly just have it in with all those on that pr.